# 请求内容为string，可选值有 on 和 off: 示是否需要展示pcd数据
string flag
---
# 0 表示更改成功，1表示失败
int16 status